Xamarin.Essentials/Screenshot/MediaFile.shared.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+
+namespace Xamarin.Essentials
+{
+    public class ScreenshotFile : ReadOnlyFile, IDisposable
```

Comment:
On this, is there a reason we wouldn't just store the image in memory as a stream? It should never be that big, then we don't need to worry about anything. The user could then save it to a file or process it right out the gate.

@jamesmontemagno Not sure if we need some sort of guidelines for when to use an in-memory stream or cache to disk first. This will come in with a camera picker or something. The image might be very big. But, I think there should always be enough memory to hold a bitmap of the screen dimensions since the OS just rendered it.

Reply:
Hmm... I think I like the file idea actually. This will make is consistent with all the other types of pickers. We probably need to rename the source code file though (from MediaFIle to ScreenshotFile)

Reply:
Yeah, the branch existed before Media file existed, there are still some remnants. Good catch!